### ISSUE TITLE
HMMUnitTest being skipped in travis

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/gatk-protected#440]
I noticed that some tests in HMMUnitTest are being skipped in travis.  Is this expected? 

```
Gradle suite > Gradle test > org.broadinstitute.hellbender.utils.hmm.HMMUnitTest.testBackward SKIPPED
Gradle suite > Gradle test > org.broadinstitute.hellbender.utils.hmm.HMMUnitTest.testFBResult SKIPPED
Gradle suite > Gradle test > org.broadinstitute.hellbender.utils.hmm.HMMUnitTest.testForward SKIPPED
Gradle suite > Gradle test > org.broadinstitute.hellbender.utils.hmm.HMMUnitTest.testPosterior SKIPPED
Gradle suite > Gradle test > org.broadinstitute.hellbender.utils.hmm.HMMUnitTest.testViterbi SKIPPED
```

**Comment by LeeTL1220** [comment]
@davidbenjamin @vruano Is this okay?  Can we activate or eliminate these tests?

**Comment by samuelklee** [comment]
Duplicate of #478, I believe this was closed by f3064de.
